### DISCUSSION TITLE
Data Access Request (DAR) is denied

### DISCUSSION THREAD

**Discussion by Junezhujun** [now-i-know-my-abcd/docs#94] [Stats & Code Help] [answered]
Hi,everyone. I am a resident doctor from China.Recently, I have been following the tutorial to apply for access to the ABCD/CCF dataset. However, upon submitting my data access request, it was immediately denied without any explanation provided.

  When I reached out to the NDA Helpdesk with my query, I received the following response: "NIH has received and processed your Data Access Request (DAR). This request is denied." My institution has an active Federal-Wide Assurance, so that should not be cause of the problem.In my search for a solution, I came across an executive order issued by the United States on February 28, 2024, which aims to prevent entities from countries of concern from accessing large volumes of sensitive personal data and government-related data of U.S. citizens.

  I am wondering if my inability to gain access could be related to this executive order. Given the immense value of the ABCD dataset to our research, we are keen on finding a way to obtain access to the database. Could you please advise on what steps I might take or whom I should contact to resolve this issue?

  Thank you very much for your time and assistance. I'm looking forward to your guidance.

**Comment by angielaird** [comment, marked as answer]
An executive order issued by the Biden Administration now prohibits the sharing of Americans' bulk personal data, including genomic data, in certain cases. You can read about it [here](https://www.whitehouse.gov/briefing-room/presidential-actions/2024/02/28/executive-order-on-preventing-access-to-americans-bulk-sensitive-personal-data-and-united-states-government-related-data-by-countries-of-concern/#%3A~%3Atext=%28c%29%20The%20term%20“country%2CStates%20or%20the%20security%20and). 

The ABCD Study has posted announcements about this issue, as well as the current problems with data downloads, on the [ABCD Wiki splash page](https://wiki.abcdstudy.org).

**Reply by Junezhujun** [reply]
Thanks for your answer！ I just saw the news on the website, which is a pity.
